homeassistant/components/neato/services.yaml

```diff
@@ -1,5 +1,9 @@
 custom_cleaning:
```

Comment:
When migrating the services to the new format, please do a complete migration. Add things like `name`, `default` and selectors.

Reply:
Will check the full list asap

Reply:
Done, but default.
Didn't find an example on documentation.

Simone

Reply:
Added them via review, for now, so they can be in the beta cut.